lib/src/hydrated_bloc_storage.dart

```diff
@@ -20,69 +24,107 @@ abstract class HydratedStorage {
   Future<void> clear();
 }
 
-/// Implementation of `HydratedStorage` which uses `PathProvider` and `dart.io`
+/// Implementation of [HydratedStorage] which uses `PathProvider` and `dart.io`
 /// to persist and retrieve state changes from the local device.
-class HydratedBlocStorage implements HydratedStorage {
-  static final _lock = Lock();
-  final Map<String, dynamic> _storage;
-  final File _file;
-
+class HydratedBlocStorage extends HydratedStorage {
   /// Returns an instance of `HydratedBlocStorage`.
-  /// `storageDirectory` can optionally be provided.
+  /// [storageDirectory] can optionally be provided.
   /// By default, `getTemporaryDirectory` is used.
+  ///
+  /// [encryptionCipher] is Hive's `HiveCipher`,
```

Comment:
can we update the docs to reflect the new `HydratedCipher`?

Reply:
yep, noticed that too